### PR TITLE
Introduce `schematic_voltage_probe`create svg function

### PR DESCRIPTION
**Pull request by AnasSarkiz**
/claim #145

Screenshot 2024-12-10 at 9 27 24 PM

### CODE REVIEW

File: lib/sch/convert-circuit-json-to-schematic-svg.ts [hunk 1]
```diff
@@ -105,7 +106,7 @@ export function convertCircuitJsonToSchematicSvg(
   const schTraceSvgs: SvgObject[] = []
   const schNetLabel: SvgObject[] = []
   const schText: SvgObject[] = []
-
+  const voltageProbeSvg: SvgObject[] = []
```
Comment (seveibar) [inline lib/sch/convert-circuit-json-to-schematic-svg.ts:109]:
```suggestion
  const voltageProbeSvgs: SvgObject[] = []
```

**Review by seveibar — COMMENTED** [review]
looks good, but a couple quick things:

- The probes are not drawn at the proper position, they should "point to" the "position"
- Can you show this with the "voltage" text too? i.e. "5V"
  - When you add the voltage text, can you add `data-schematic-voltage-probe-id` to it? We will be dynamically altering it when running simulations

**Comment by AnasSarkiz** [comment]
@seveibar 
Screenshot 2024-12-11 at 10 06 25 PM

**Comment by seveibar** [comment]
@AnasSarkiz looks good- can you cut a bit off the line?

image

**Comment by AnasSarkiz** [comment]
Screenshot 2024-12-11 at 10 09 21 PM

**Comment by AnasSarkiz** [comment]
> Screenshot 2024-12-11 at 10 09 21 PM

@seveibar  take a look

**Comment by AnasSarkiz** [comment]
@seveibar  check the last update.
We can add an underline to the text if you need it!

example:

Screenshot 2024-12-12 at 11 29 40 AM

**Review by seveibar — APPROVED** [review]

**Pull request merged**